src/main/scala/wdl4s/expression/FileEvaluator.scala

```diff
@@ -55,6 +55,8 @@ case class FileEvaluator(valueEvaluator: ValueEvaluator, coerceTo: WdlType = Wdl
         a.value.flatMap(findWdlFiles(_, coerce=false))
       case Success(m: WdlMap) =>
         m.value flatMap { case (k, v) => Seq(k, v) } flatMap(findWdlFiles(_, coerce=false)) toSeq
+      case Success(WdlOptionalValue(_, Some(v))) => findWdlFiles(v)
+      case Success(WdlPair(l, r)) => findWdlFiles(l) ++ findWdlFiles(r)
```

Comment:
ToL: I don't know much about this method. Why do the above set `coerce = false`, but these leave `coerce = true`?